node/ChainwebNode.hs

```diff
@@ -449,7 +450,14 @@ mainInfo = programInfoValidate
     (defaultChainwebNodeConfiguration Testnet02)
     validateChainwebNodeConfiguration
 
+-- | KILLSWITCH: The logic surrounding `txSilenceDates` here is to be removed in
+-- a future version of Chainweb. This prevents the Node from even starting if
+-- past a specified date.
+--
 main :: IO ()
-main = withWatchdog $ runWithPkgInfoConfiguration mainInfo pkgInfo $ \conf -> do
+main = withWatchdog . runWithPkgInfoConfiguration mainInfo pkgInfo $ \conf -> do
     let v = _configChainwebVersion $ _nodeConfigChainweb conf
-    withNodeLogger (_nodeConfigLog conf) v $ node conf
+    now <- getCurrentTimeIntegral
+    case txSilenceDates v of
+        Just (_, end) | now > end -> pure ()
```

Comment:
I argue that running nodes should be left alive, since:

1. They will no longer be able to produce new Cuts anyway
2. They're still useful to keep around as p2p/cut sync nodes
3. If all nodes in the network suddenly died, it could affect overall stability in a bad way.

Reply:
I agree with @fosskers here, though, these useless nodes might make consensus needlessly complex